activestorage/test/analyzer/image_analyzer_test.rb

```diff
@@ -29,4 +29,13 @@ class ActiveStorage::Analyzer::ImageAnalyzerTest < ActiveSupport::TestCase
     assert_equal 792, metadata[:width]
     assert_equal 584, metadata[:height]
   end
+
+  test "analyzing an unsupported image type returns nil width and height metadata" do
```

Comment:
Name this test “analyzing an unsupported image type” to match the others in this file. The test itself is clear as to what outcome it expects.